.pytool/CISettings.py

```diff
@@ -189,17 +187,17 @@ def GetDependencies(self):
             {
                 "Path": "MU_BASECORE",
                 "Url": "https://github.com/microsoft/mu_basecore.git",
-                "Branch": "dev/202405"
+                "Branch": "release/202502"
             },
             {
                 "Path": "Common/MU_PLUS",
                 "Url": "https://github.com/microsoft/mu_plus.git",
-                "Branch": "dev/202405"
+                "Branch": "release/202502"
             },
             {
                 "Path": "Common/MU_TIANO_PLUS",
                 "Url": "https://github.com/microsoft/mu_tiano_plus.git",
-                "Branch": "dev/202405"
+                "Branch": "release/202502"
```

Comment:
We want to keep these on dev, right?

Reply:
We want feature repos to be on release branch, because that is what they will be paired with in projects.

We want to have overridetracking use the --track tag and maintain an override for both dev and release branches.